lib/content/audio/no_service_to_destination.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Content.Audio.NoServiceToDestination do
+  @moduledoc """
+  No service to [destination]
+  """
+
+  require Logger
+
+  @enforce_keys [:message]
+  defstruct @enforce_keys
+
+  @type t :: %__MODULE__{
+          message: String.t()
+        }
+
+  def from_message(
+        %Content.Message.Alert.DestinationNoService{destination: destination} = message
+      ) do
+    case PaEss.Utilities.destination_to_ad_hoc_string(destination) do
+      {:ok, destination} ->
+        audio = "No service to #{destination}"
+
+        %__MODULE__{
+          message: audio
+        }
+
+      {:error, :unknown} ->
+        Logger.error(
+          "NoServiceToDestination.from_message unknown destination: #{inspect(message)}"
+        )
+
+        nil
+    end
+  end
+
+  def from_message(%Content.Message.Alert.NoServiceUseShuttle{destination: destination} = message) do
+    case PaEss.Utilities.destination_to_ad_hoc_string(destination) do
+      {:ok, destination} ->
+        audio = "No service to #{destination} use shuttle"
+
+        %__MODULE__{
+          message: audio
+        }
+
+      {:error, :unknown} ->
+        Logger.error(
+          "NoServiceToDestination.from_message unknown destination: #{inspect(message)}"
+        )
+
+        nil
+    end
+  end
+
+  defimpl Content.Audio do
+    def to_params(%Content.Audio.NoServiceToDestination{message: message}) do
+      {:ad_hoc, {message, :audio}}
```

Comment:
I don't think we have any existing audio files that would make a for a clear audio message so I'm going with ad_hoc for now. This means that it will use the ARINC robo-voice which is not ideal, but is better than nothing. It would be good though to follow up on this and have ARINC help us set up message IDs for this type of message where we can pass the destination as a parameter.